runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerLog.java

```diff
@@ -40,13 +43,29 @@
 import net.runelite.client.util.StackFormatter;
 
 @Getter
-class LootTrackerBox extends JPanel
+class LootTrackerLog extends JPanel
 {
 	private static final int ITEMS_PER_ROW = 5;
-	private final long totalPrice;
 
-	LootTrackerBox(final ItemManager itemManager, final String title, final String subTitle, final LootTrackerItemEntry[] items)
+	private final JPanel itemContainer = new JPanel();
+	private final JLabel priceLabel = new JLabel();
+	private final JLabel subTitleLabel = new JLabel();
+
+	private final ItemManager itemManager;
+
+	private long totalPrice;
+	private int totalKills;
+
+	@Getter
+	private String title;
```

Comment:
Just override equals and compare 2 lootTrackerBoxes instead of exposing title just for that

Reply:
Also, rename this to id or something

Reply:
Won't I have to expose the id then? to compare between two boxes

Reply:
Nope, you would compare them based on title. `@EqualsAndHashCode(of='title')` or something like that

Reply:
Ah, didn't know about that annotation

Reply:
Still, why would I compare two boxes if I don't need to create a second in the first place, I can just search for that ID, and if a box with that Id already exists, add items to it, instead of creating a new box and comparing it, only to then throw it away